lib/modules/versioning/same-major/index.spec.ts

```diff
@@ -0,0 +1,50 @@
+import sameMajor from '.';
+
+describe('modules/versioning/same-major/index', () => {
+  describe('.matches(version, range)', () => {
+    it('should return true when version is has same major', () => {
+      expect(sameMajor.matches('1.0.1', '1.0.0')).toBeTrue();
+    });
+
+    it('should return false when version has different major', () => {
+      expect(sameMajor.matches('2.0.1', '1.0.0')).toBeFalse();
+    });
+
+    it('should return false when version is lesser than the current version', () => {
```

Comment:
```suggestion
    it('should return false when version is out of range', () => {
```
Add some more assertions